source/base/Text.ooc

```diff
@@ -53,6 +53,30 @@ Text: cover {
 	operator != (other: String) -> Bool { !(this == other) }
 	operator != (other: This) -> Bool { !(this == other) }
 	operator + (other: This) -> This { This new(this _buffer + other _buffer) }
+	operator + (other: Int) -> This {
+		converted := other toString()
```

Comment:
Huh, actually, this does not work for me:
```ooc
extend Float {
	toText: func -> Text {
		converted := this toString()
		result := Text new(converted)
		converted free()
		result
	}
}

operator + (other: Float) -> This {
	this + other toText()
}
```
I get all sorts of weird characters in the resulting text instead. If I don't free `converted`, it works, but then it of course leaks memory. I don't get the same problem for `Int`.